opcua/binary_client.py

```diff
@@ -10,21 +10,12 @@
 import opcua.uaprotocol as ua
 import opcua.utils as utils
 
-
-class CachedRequest(object):
```

Comment:
Yes, this class was only needed for wrapping binary data and passing it to a function that built requests from objects. Now another function is used, it expects just binary data.